src/internal/operators/timeInterval.ts

```diff
@@ -51,16 +51,18 @@ import { map } from './map';
  * value and interval.
  */
 export function timeInterval<T>(scheduler: SchedulerLike = async): OperatorFunction<T, TimeInterval<T>> {
-  return (source: Observable<T>) => defer(() => {
-    return source.pipe(
-      // TODO(benlesh): correct these typings.
-      scan(
-        ({ current }, value) => ({ value, current: scheduler.now(), last: current }),
-        { current: scheduler.now(), value: undefined,  last: undefined } as any
-      ) as OperatorFunction<T, any>,
-      map<any, TimeInterval<T>>(({ current, last, value }) => new TimeInterval(value, current - last)),
-    );
-  });
+  return (source: Observable<T>) =>
+    defer(() => {
+      return source.pipe(
+        // TODO(benlesh): correct these typings.
+        scan(({ current }, value) => ({ value, current: scheduler.now(), last: current }), {
+          current: scheduler.now(),
+          value: undefined,
+          last: undefined,
+        } as any) as OperatorFunction<T, any>,
+        map<any, TimeInterval<T>>(({ current, last, value }) => new TimeInterval(value, current - last))
+      );
+    });
```

Comment:
This was done as part of the automatic pre commit hook. I can remove it if you would only like to include the fix within the JSDoc comment.